.travis.yml

```diff
@@ -121,33 +129,27 @@ before_script:
       --project phenomenal-1145 -q
 
 script:
-  # Packer -force for Azure and Openstack does not automatically remove the artifacts from previous build.
+  # Finally bulding the image with packer
+  - travis_retry /tmp/packer build -force build-"$HOST_CLOUD".json | tee /tmp/pckr_build_log.txt
+
+after_success:
+  # Azure and Openstack packer builder do not automatically remove namesake artifacts from previous builds.
+  # They allows two image with same names, but different unique IDs, even with packer flag --force
+  # Thus after creation of either a new or an updated kubenow image, we will remove previous old ones
   - >
     if [ "$HOST_CLOUD" = 'azure' ]; then
         bin/clean_az_dupl.sh
     elif [ "$HOST_CLOUD" = 'openstack' ]; then
         bin/clean_os_dupl.sh
     else
-        echo "Variable HOST_CLOUD not set."
-    fi
-
-  # Check whether this build was triggered by our cron job for security-updates
-  - echo "TRAVIS_EVENT_TYPE is:$TRAVIS_EVENT_TYPE"
-  - >
-    if [ "$TRAVIS_EVENT_TYPE" = 'cron' ]; then
```

Comment:
This check needs to be done before running the packer build command in order to correctly provision the builder with either the requirement script or the one about checking security updates.